sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -29,206 +29,92 @@ When the Azure AD authentication events service calls your custom extension, it
 
-An example `local.settings.json` file:
+### Authentication events trigger output binding
 
-```json
-{
-  "IsEncrypted": false,
-  "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "AuthenticationEvents__TenantId": "8615397b-****-****-****-********06c8",
-    "AuthenticationEvents__AudienceAppId": "api://46f98993-****-****-****-********0038",
-    "AuthenticationEvents__CustomCallerAppId": "46f98993-****-****-****-********0038"
-  }
-}
-```
+The authentication events trigger output binding allows a function to send authentication event actions to the Azure AD event service.
 
-#### No token validation
+## Documentation
 
-If you would like to _not_ authenticate the token while in local development, set the following application settings in the [local.settings.json](https://learn.microsoft.com/azure/azure-functions/functions-develop-local#local-settings-file) file:
+Learn about custom [claims providers and custom extensions](https://learn.microsoft.com/en-us/azure/active-directory/develop/custom-claims-provider-overview).  Learn how to create and register your custom claims provider API, how to add tokens from external stores, and how to test and troubleshoot your API.
 
-* **AuthenticationEvents__BypassTokenValidation** - value of `true` will make the trigger not check for a validation of the token.
+## Examples
 
 ### Quickstart
 
-* Visual Studio 2019
-  * Start Visual Studio
-  * Select "Create a new project"
-  * In the template search area search and select "AzureAuthEventsTrigger"
-  * Give your project a meaningful Project Name, Location, Solution and Solution Name.
-
-* Visual Studio Code
-  * Start Visual Studio Code
-  * Run the command "Create Azure Authentication Events Trigger Project" via the command palette
-  * Follow the project creation prompts
-* Please note: that on a first time run it might take awhile to download the the required packages.
-* For development purpose turn of token validation for testing:
-* Add the **AuthenticationEvents__BypassTokenValidation** application key to the "Values" section in the local.settings.json file and set it's value to **true**. If you do not have a local.settings.json file in your local environment, create one in the root of your Function App.
-
-```json
-{
-  "IsEncrypted": false,
-  "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "AuthenticationEvents__BypassTokenValidation": true
-  }
-}
-```
-
-* Once the project is loaded, you can run the sample code and you should see the Azure functions developer's application load your end point.
+#### Prerequisites
 
-## Key concepts
+1. [Download Postman](https://www.postman.com/downloads/)
```

Comment:
To avoid looking like a Postman endorsement, should we consider mentioning alternatives such as Fiddler, Fork, et al?     Perhaps mention that these examples will use Postman for demonstrative purposes, but any would work?

Reply:
Agreed with this, we can rather give them just the payloads and give developers an option of tools, we can still keep postman in the wiki rather...

Reply:
That's fine.  I could use some example payloads using the new data contract, though.  Will follow up with Shane on this.

Reply:
I added some more generic instructions on sending requests.  Still need to update the payload example in the future with the newer data contracts (the older version should still be supported, I believe.